tests/cpydiff/core_class_name_mangling.py

```diff
@@ -0,0 +1,25 @@
+"""
+categories: Core,Classes
+description: Private Class Members name mangling is not implemented
+cause: THe MicroPython compiler has not yet implemented name mangling for private class members at runtime.
+workaround: Avoid using or having a collision with global names, by adding a unique prefix to the private class member name manually.
+"""
+
+
+def __print_string(string):
+    print(string)
+
+class Foo:
+    def __init__(self, string):
+        self.string = string
+    
+    def do_print(self):
+        __print_string(self.string)
+
+example_string = "Example String to print."
+print(example_string)
+
+class_item = Foo(example_string)
+print(class_item.string)
+
+class_item.do_print()
```

Comment:
Do we need these first two `print()`s since they are not demonstrating the difference?

```suggestion
class_item = Foo("Example String to print.")
class_item.do_print()
```

Reply:
The first print can definitely be removed. I added the `print(class_item.string)` to show the attribute for the instance is usable, and not part of the issue. It isn't needed, but to me it provides clarity, when reading the generated documentation, if you don't already know what name mangling is? 

Since it is just showing it is a usable property, is there a better way to show that, that doesn't involve printing it, that would make that idea more clear, if it should be kept at all?

Reply:
> It isn't needed, but to me it provides clarity, when reading the generated documentation, if you don't already know what name mangling is?

I already know what name mangling is, so I'm probably not the best person to ask. :smile: 

> is there a better way to show that, that doesn't involve printing it, that would make that idea more clear, if it should be kept at all?


I think using `print()` is a good way to show the "did not raise exception" case and is consistent with other CPy diff snippets.

Reply:
I think just remove the first `print`.

Reply:
I removed the first print.